graylog2-web-interface/src/views/components/searchbar/StreamsFilter.tsx

```diff
@@ -33,9 +33,10 @@ type Props = {
   value: Array<string>,
   streams: Array<{ key: string, value: string }>,
   onChange: (newStreamIds: Array<string>) => void,
+  multi?: boolean,
 };
 
-const StreamsFilter = ({ disabled, value, streams, onChange }: Props) => {
+const StreamsFilter = ({ disabled, value, streams, onChange, multi }: Props) => {
```

Comment:
Why do we need this change in this PR?

Reply:
This is necessary for making the streams filter reusable and I did not want to create another PR for this small change.